internal/admin/templates/layout.html

```
@@ -24,47 +24,128 @@
+      {{- /* Pending-pairing count rides the Devices entry instead of the
+          old header slot: it is a badge ON the destination, which is
+          where a count belongs, and it survives the sidebar having no
+          top-right meta strip to hang from. app.js resolves it by id and
+          only toggles [hidden] + the count text, so the move is CSS-only
+          from its side. */ -}}
+      <span id="pairing-badge" class="pairing-badge" hidden aria-label="Pending pairing requests">
+        <span class="pairing-badge-dot" aria-hidden="true"></span>
+        <span class="pairing-badge-count" data-count="0"
+              role="status" aria-live="polite" aria-atomic="true">0</span>
+        <span class="pairing-badge-label">pending</span>
+      </span></a>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </a> ]

(tag-pair)